### ISSUE TITLE
[BUG] Confirmation link validity / time zone

### ISSUE THREAD

**Issue by sthuis** [derhansen/sf_event_mgt#175]
Setting a value for plugin.tx_sfeventmgt.settings.confirmation.linkValidity does not have any effect. Confirmation link validity always remains at the default value of 1h. 
The text in the registration mail to the user that informs about the expiry time of the confirmation link does not adapt to the correct time zone but shows the UTC value.

**Comment by derhansen** [comment]
Thanks for your report.

I fixed the problem with the linkValidity in the current master branch and a bugfix update on TER will follow soon.

I could not reproduce the problem with the timezone. The expiry time is generated serverside and depends on the webservers date/time settings. Do you have any special settings in [SYS][phpTimeZone]?

**Comment by sthuis** [comment]
Thank you very much for the quick fix. I just updated to the master and it works perfectly now.

Regarding the time zone issue: [SYS][phpTimeZone] was empty and the time zone was not set on the web server. I set the correct value and everything works as expected now.

I'm very satisfied with this extension and will use it in my web project now. It will serve for registration for a sports event. Will send you a link as soon as the website is finished.
